CONTRIBUTING.md

```diff
@@ -45,7 +45,7 @@ Once the fonts are downloaded, add them to the Xcode project under _Example for
 
 ## Testing
 
-Tests can be run as usual from Xcode(Product -> Test or cmd+U). Snapshot tests should be run on an iPhone 8 running iOS 13.5 to match what is used on CI.
+Tests can be run as usual from Xcode(Product -> Test or cmd+U). Snapshot tests should be run on an iPhone 8 running iOS 14.4 to match what is used on CI.
```

Comment:
To avoid this in the future, what if we say
```suggestion
Tests can be run as usual from Xcode(Product -> Test or cmd+U). Snapshot tests should be run on the [device specified for CI](https://github.com/Skyscanner/backpack-ios/blob/3b23a51b85a9dc778599c4eab21a8b8b44903cb8/.github/workflows/ci.yml#L132).
```

Reply:
Or maybe we should just put a comment somewhere to say if you're changing it, we need to update CONTRIBUTING.md too?

Reply:
I think the link might be okay but it should link to the file on main rather than a specific commit

Reply:
Yep indeed! I did notice that one!